probe/endpoint/conntrack.go

```diff
@@ -75,11 +81,11 @@ type Conntracker struct {
 }
 
 // NewConntracker creates and starts a new Conntracter
-func NewConntracker(existingConns bool, args ...string) (*Conntracker, error) {
+var NewConntracker = func(existingConns bool, args ...string) (Conntracker, error) {
```

Comment:
As a general statement, I think we've passed the threshold where we're overusing package-level construction mocks/stubs. I'd like to issue a moratorium on the creation of new stubs, and instead pass factory functions to components that need them as parameters. What do you think?